### PR TITLE
Nim is ready for CI

### PR DESCRIPTION
**Pull request by sealmove**
`ci-nim` produces a correct `ci.json` in `$TEST_OUT_DIR/nim`.
This PR goes together with [this one](https://github.com/kaitai-io/kaitai_struct_compiler/pull/170).
Can we merge both so I can proceed with setting up Nim for Travis in [ci_targets repo](https://github.com/kaitai-io/ci_targets)?

### CODE REVIEW

File: run-nim [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+. ./config
+
+if [ ! -f "spec/nim/test.nim" ]; then
+  RED='\033[1;31m'
```
Comment (GreyCat) [inline run-nim]: The rest of these shell scripts actually use tabs for indentations, can I ask you to replace spaces here with tabs for consistency?

File: run-nim [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+mkdir -p "$TEST_OUT_DIR/nim/bin"
+
+i=0 args=()
+for var in "$@"
+do
+  args[$i]="Kaitai Struct Compiler Test Suite::Nim: $var"
+  ((++i))
+done
+
+echo $arguments
```
Comment (GreyCat) [inline run-nim]: There is no `$arguments` variable in this script, only `$args`, am I missing something?

File: run-nim [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  exit
+fi
+
+rm -rf "$TEST_OUT_DIR/nim"
+mkdir -p "$TEST_OUT_DIR/nim/bin"
+
+i=0 args=()
+for var in "$@"
+do
+  args[$i]="Kaitai Struct Compiler Test Suite::Nim: $var"
```
Comment (GreyCat) [inline run-nim]: These are arrays, they won't work in /bin/sh. Either use /bin/bash, or use a more shell-like approach, such as processing arguments as individual lines with `sed 's/^/Kaitai Struct Compiler Test Suite::Nim: /'`

File: run-nim (hunk 1 above)
Comment (sealmove) [reply run-nim]: Sure, my bad for not noticing.

File: run-nim (hunk 2 above)
Comment (sealmove) [reply run-nim]: Leftover from testing the script, I'll remove it.

File: run-nim (hunk 3 above)
Comment (sealmove) [reply run-nim]: I changed the shebang to /bin/bash for now, but if /bin/sh is preferred feel free to me.

**Comment by GreyCat** [comment]
LGTM, thanks, merging in!

**Pull request merged**